project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+      - &foundation_wall_insulation_finished_basement
+        option: Insulation Foundation Wall|Wall R-10, Interior
+        apply_logic: 
+          - and:
+            - Geometry Foundation Type|Heated Basement
+            - Insulation Foundation Wall|Uninsulated
+        costs:
+          - value: 4.56 # $/ft2 (3.54* $/ft2 * 1.289 inflation) based on cost in BEopt
+            multiplier: Wall Area, Below-Grade (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf        
+      - &rim_joist_insulation
+        option: Insulation Rim Joist|R-10, Exterior
+        apply_logic: 
+          - and:
+            - or:
+              - Geometry Foundation Type|Unvented Crawlspace
+              - Geometry Foundation Type|Vented Crawlspace
+              - Geometry Foundation Type|Heated Basement
+            - Insulation Foundation Wall|Uninsulated
+        costs:
+          - value: 2.32 # $/ft2 (1.8* $/ft2 * 1.289 inflation) https://remdb.nrel.gov/measures.php?gId=14&ctId=378&scId=5929&acId=5932)
+            multiplier: Rim Joist Area, Above-Grade, Exterior (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #seal crawlspace vents
+      - &seal_vented_crawlspaces
+        option: Geometry Foundation Type|Unvented Crawlspace
+        apply_logic:
+          and:
+            - Geometry Foundation Type|Vented Crawlspace
+            - or: # Ensure that the dwelling unit being modeled is either (1) not in a multifamily building, or (2) on the bottom floor of a multifamily building and therefore above the crawlspace.
+                - Geometry Building Level MF|None
+                - Geometry Building Level MF|Bottom
+        costs:
+          - value: 1.083 # $/ft2 (0.84* $/ft2 * 1.289 inflation) https://remdb.nrel.gov/measures.php?gId=14&ctId=334&scId=5169&acId=6565)
+            multiplier: Floor Area, Foundation (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #insulate finished attics and cathedral ceilings to R-30
+  - upgrade_name: Advanced Enclosure
+    #Up
+    options:
+      #Light Enclosure Measures
+      - *attic_insulation_IECC_CZ1A
+      - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      - *infiltration_30pct_reduction
+      #Intermediate Enclosure Measures
+      - *ducts_category1
+      - *ducts_category2
+      - *ducts_category3
+      - *ducts_category4
+      #- *ducts_category5
+      #- *ducts_category6
+      - *ducts_category7
+      - *Drill-and-fill
```

Comment:
Do we want to extend drill-and-fill for Wood Stud, R-7 and R-11?

Reply:
The intent document only focus on dwelling units with no insulation and wood stud walls

Reply:
@ekpresent

Reply:
Drill and fill for uninsulated wood stud walls only (my understanding is it's more complicated/expensive/invasive if there is existing insulation).